grc/blocks/variable_function_probe.block.yml

```diff
@@ -31,12 +31,23 @@ templates:
         import time
         import threading
     var_make: self.${id} = ${id} = ${value}
-    make: "\ndef _${id}_probe():\n    while True:\n        <% obj = 'self' + ('.'\
-        \ + block_id if block_id else '') %>\n        val = ${obj}.${function_name}(${function_args})\n\
-        \        try:\n            self.set_${id}(val)\n        except AttributeError:\n\
-        \            pass\n        time.sleep(1.0 / (${poll_rate}))\n_${id}_thread\
-        \ = threading.Thread(target=_${id}_probe)\n_${id}_thread.daemon = True\n_${id}_thread.start()\n\
-        \    "
+    make: |+
+
```

Comment:
extra \n here

Reply:
I'm not sure what you mean. Do you want me to remove the empty line 35, or to add "\n" there?

Reply:
remove the empty line; it is not necessary & doesn't match up with the formatting in the rest of the file

Reply:
That's what I thought but I wanted to make sure. It's done.

Reply:
:+1: thx!